oh_staff_ui/models.py

```diff
@@ -17,3 +18,34 @@ class ItemType(models.Model):
 
     def __str__(self):
         return self.type
+
+
+class ProjectItem(models.Model):
+
+    ark = models.CharField(max_length=40, blank=False, null=False)
+    create_date = models.DateField()
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name="+",
+    )
+    last_edit_date = models.DateField()
```

Comment:
@kjallen For consistency, please rename `last_edit_date` to `last_modified_date`.  This change will need to be made in `admin.py` and in the sample data as well.